script.colorbox/resources/lib/Utils.py

```diff
@@ -20,23 +20,25 @@
 ADDON_ID =          ADDON.getAddonInfo('id')
 ADDON_LANGUAGE =    ADDON.getLocalizedString
 ADDON_DATA_PATH =   os.path.join(xbmc.translatePath("special://profile/addon_data/%s" % ADDON_ID))
-ADDON_COLORS =      os.path.join(ADDON_DATA_PATH, "colors.txt")
+ADDON_COLORS =      os.path.join(ADDON_DATA_PATH, "colors.db")
```

Comment:
Are you writing tables or plain text? If plan text; its advisable to leave the text extension, However IMO this is not a deal breaker :)